recipes/slow-buffer-to-buffer-alloc-unsafe-slow/codemod.yaml

```diff
@@ -0,0 +1,23 @@
+schema_version: "1.0"
+name: "@nodejs/slow-buffer-to-buffer-alloc-unsafe-slow"
+version: "1.0.0"
+description: Handle DEP0030 via transforming SlowBuffer usage to Buffer.allocUnsafeSlow().
+author: lluisemper(Lluis Semper Lloret)
+license: MIT
+workflow: workflow.yaml
+category: migration
+
+targets:
+  languages:
+    - javascript
+    - typescript
+
+keywords:
+  - transformation
+  - migration
+  - buffer
+  - slowbuffer
```

Comment:
@alexbit-codemod could you confirm that this keywords is good idea for ours codemods ??? The published codemods only use `transformation` & `migration` keywords.

Reply:
Sorry! Just saw this! and I know it’s already approved so I don’t want to block it. The feature is nice to have as is, but if you want to make it more useful, here’s a guide for applying tags:
https://go.codemod.com/tags-guide

Tags are optional, but they’ll show up in the codemod registry so users can quickly tell what the codemod is about. Words like `transformation` or `migration` aren’t very helpful. cc @lluisemper @AugustinMauroy